### DISCUSSION TITLE
Which notice shall OPT-100-Contract Framework Notice Identifier refer to?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#300] [Q&A] [answered]
Hi,
Which notice shall OPT-100-Contract Framework Notice Identifier refer to? Shall it refer to the previously published Contract notice (or in some cases Prior information notice with call for competition)?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This field is used for contracts awarded within a Framework Agreement, the field should then refer to the published Contract Award Notice informing about the Framework Contract signature.
For non-technical questions, please send these directly to the helpdesk: ted@publications.europa.eu
